packages/core/admin/server/src/controllers/admin.ts

```diff
@@ -154,12 +154,18 @@ export default {
   async plugins(ctx: Context) {
     const enabledPlugins = strapi.config.get('enabledPlugins') as any;
 
-    const plugins = Object.entries(enabledPlugins).map(([key, plugin]: any) => ({
-      name: plugin.info.name || key,
-      displayName: plugin.info.displayName || plugin.info.name || key,
-      description: plugin.info.description || '',
-      packageName: plugin.info.packageName,
-    }));
+    // List of core plugins that are always enabled,
+    // and so it's not necessary to display them in the plugins list
+    const CORE_PLUGINS = ['content-manager', 'content-type-builder', 'email', 'upload', 'i18n'];
+
+    const plugins = Object.entries(enabledPlugins)
```

Comment:
It may not be feasible but I wonder if core packages should be enabled in `strapi.config.get('enabledPlugins')` at all?

Reply:
Originally I was going this route, of not including them in the enabled plugins config,  but those are also used to load the pacakges itself, removing them from that config would mean refactoring how we load plugins, which felt like a lot of work